src/services/ResourceProvider/ResourceProvider.ts

```diff
@@ -0,0 +1,5 @@
+export default class ResourceProviderService {
+    get(resourceName: string): void {
```

Comment:
```suggestion
    get(resourceName: string): string {
```

`string` powinno przejść, a jak nie to zostaw void

Reply:
zrobione